lib/fizzy/execute.cpp

```diff
@@ -299,6 +299,19 @@ inline uint64_t popcnt64(uint64_t value) noexcept
     return static_cast<uint64_t>(__builtin_popcountll(value));
 }
 
+template <typename T>
+T signbit(T value) noexcept = delete;
+
+inline bool signbit(float value) noexcept
+{
+    return (bit_cast<uint32_t>(value) & F32SignMask) != 0;
```

Comment:
This generates two very simple instructions with GCC, but clang detects this pattern and replaces it with the SSE instruction.